EjerciciosDiaUno.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+public class Class1
+{
+	public Class1()
+	{
+        int Resultado = 0;
+
+        static void Main(string[] args)
+        {
+            EjercicioNumeroUno calculo = new EjercicioNumeroUno();
+
+            System.Console.WriteLine("La Suma es: " + calculo.CalcularResultado(24, 40));
+
+            Convert.ToString(calculo, null);
+
+            System.Console.WriteLine("La Multiplicación es: " + calculo.CalcularResultado(24, 40, 2));
+
+            Console.ReadKey();
+        }
+
+        int CalcularResultado(int number1, int number2)
```

Comment:
1) El enunciado del documento dice **devuelva en String el resultado**
2) En el documento dice. **La clase debe ser pública al igual que todos sus métodos**
3) Se podria cambiar las variables solo texto directamente en vez de usar numero en el
4) Si se va a manejar todo en ingles, DEBE SER TODO EN INGLES, o si lo quiere manejar todo en español DEBE SER TODO EN ESPAÑOL, no mezclar idiomas

```suggestion
       public  String CalcularResultado(int numeroUno, int numeroDos)
```

Reply:
1. No iniciare un ejercicio hasta no comprenderlo a cabalidad.
2. Reestructurare el nombramientos de clases, métodos y variables, y también nombrare a todos los métodos públicos.
3. Acataré tu consejo..
4. Trabajaré en un solo idioma.